ui/src/views/DNSRecords.vue

```diff
@@ -293,7 +293,12 @@ export default {
       this.loading.setDNSrecords = false;
 
       for (const validationError of validationErrors) {
-        this.error.address = this.$t("error." + validationError.error);
+        if (validationError.field.split(".")[2] === "address") {
```

Comment:
The result of this `split` has length = 1 in some cases. It is safer to get the field of `validationError` like this: https://github.com/NethServer/ns8-core/blob/main/core/ui/src/components/backup/AddRepositoryModal.vue#L1141-L1145